azure/dns.tf

```diff
@@ -3,55 +3,55 @@ data "azurerm_dns_zone" "hosted" {
 }
 
 resource "azurerm_dns_a_record" "opsmanager" {
-  name                = "opsmanager"
+  name                = "opsmanager.${var.environment_name}"
```

Comment:
I think I see where the confusion was coming from with the absence of these.  My hosted zone includes the `environment_name` in it already, so by putting this in here I end up with duplications.

Reply:
I am was aiming for the consistency.
The AWS and GCP both have the `opsmanager` + `env_name`.

Reply:
![Screenshot from 2020-01-16 16-32-02](https://user-images.githubusercontent.com/1326605/72564460-c5dec580-387d-11ea-9c99-1a988b149738.png)

Reply:
Is it? Please bring up the discussion in slack. I'm willing to bet this is one of those, _it depends on the customer_, scenarios.